### PR TITLE
Medical related fixes

### PR DESCRIPTION
**Pull request by SpadesNeil**
- Fixes https://github.com/PolarisSS13/Polaris/issues/1892 by undoing a change made by https://github.com/PolarisSS13/Polaris/commit/f75d509e7fe8e364d5db5782a3dd1ced183a1dfd

### CODE REVIEW

**Comment by Anewbe** [comment]
Pictures for mapping changes would be appreciated.

**Comment by SpadesNeil** [comment]
capture

**Comment by Anewbe** [comment]
Is that rack gone or present?

**Comment by SpadesNeil** [comment]
It's still there! Right next to the APC.

**Comment by SpadesNeil** [comment]
Hang on let me check something before we merge this.

File: code/game/objects/random/random.dm [hunk 1]
```diff
@@ -11,11 +11,9 @@
 	..()
 	if (!prob(spawn_nothing_percentage))
 		spawn_item()
-
-/obj/random/initialize()
```
Comment (SpadesNeil) [inline code/game/objects/random/random.dm]: I'm told that apparently initialize() is supposed to be called by the base New() proc, but for some reason that isn't happening with these items. My way works, the old way doesn't work, and I don't know why.

**Comment by Anewbe** [comment]
Had some discussion, was decided that Medical doesn't need a cycler there, seeing as there's already one in EVA.

**Comment by Yoshax** [comment]
Don't need the suits there either, they should be in EVA.

**Comment by SpadesNeil** [comment]
Okay gonna remove the cycler. Can remove the suits later.

**Pull request merged**